synapse/storage/engines/__init__.py

```diff
@@ -28,11 +27,8 @@ def create_engine(database_config) -> BaseDatabaseEngine:
         return Sqlite3Engine(sqlite3, database_config)
 
     if name == "psycopg2":
-        # pypy requires psycopg2cffi rather than psycopg2
-        if platform.python_implementation() == "PyPy":
-            import psycopg2cffi as psycopg2  # type: ignore
-        else:
-            import psycopg2  # type: ignore
+        # note: pypy is still supported by using psycopg2cffi-compat to monkey-patch imports to psycopg2
```

Comment:
It is here in case there is confusion with a "wait, `psycopg2` is installed?" question whenever the `python_dependencies.py` is examined and cross-referenced with this, an assurance that "`psycopg2`" is able to be properly imported on pypy after all.

Reply:
```suggestion
        # Note that psycopg2cffi-compat provides the psycopg2 module on mypy.
```